lib/flatpak-game.lua

```diff
@@ -375,24 +377,28 @@ function save_manifest(metadata, enable_network)
 end
 
 function build_export()
+    print('Building repository for ' .. metadata.id)
```

Comment:
If it doesn't print the name in the library, should it be passed back to the calling script somehow? How does the user know which game was newly added to the repository? Does this program expect the user to grep through `flatpak remote-ls` output?